code/game/objects/items/twohanded.dm

```diff
@@ -962,3 +963,167 @@
+	righthand_file = 'icons/mob/inhands/weapons/hammers_righthand.dmi'
+	name = "Vxtvul Hammer"
+	desc = "A relict sledgehammer with charge packs wired to two blast pads on its head. \
+			While wielded in two hands, the user can charge a massive blow that will shatter construction and hurl bodies."
+	force = 4 //It's heavy as hell
+	force_wielded = 24 
+	armour_penetration = 50 //Designed for shattering walls in a single blow, I don't think it cares much about armor
+	throwforce = 18
+	attack_verb = list("attacked", "hit", "struck", "bludgeoned", "bashed", "smashed")
+	sharpness = SHARP_NONE //Blunt, breaks bones
+	wound_bonus = -10
+	bare_wound_bonus = 15
+	max_integrity = 200
+	resistance_flags = ACID_PROOF | FIRE_PROOF
+	w_class = WEIGHT_CLASS_HUGE
+	slot_flags = ITEM_SLOT_BACK
+	actions_types = list(/datum/action/item_action/charge_hammer)
+	var/datum/effect_system/spark_spread/spark_system //It's a surprise tool that'll help us later
+	var/charging = FALSE
+	var/supercharged = FALSE
+
+/obj/item/twohanded/vxtvulhammer/Initialize() //For the sparks when you begin to charge it
+	. = ..()
+	spark_system = new
+	spark_system.set_up(5, 0, src)
+	spark_system.attach(src)
+
+/obj/item/twohanded/vxtvulhammer/Destroy() //Even though the hammer won't probably be destroyed, Ever™
+	QDEL_NULL(spark_system)
+	return ..()
+
+/obj/item/twohanded/vxtvulhammer/update_icon()
+	icon_state = "vxtvul_hammer[wielded]-[supercharged ? 1 : 0]"
+
+/obj/item/twohanded/vxtvulhammer/examine(mob/living/carbon/user)
+	. = ..()
+	if(supercharged)
+		. += "<b>Electric sparks</b> are bursting from the blast pads!"
+
+/obj/item/twohanded/vxtvulhammer/unwield(mob/living/carbon/user)
+	..()
+	if(supercharged) //So you can't one-hand the charged hit
+		to_chat(user, span_notice("Your hammer loses its power as you adjust your grip."))
+		user.visible_message(span_warning("The sparks from [user]'s hammer suddenly stop!"))
+		supercharge()
+	if(charging) //So you can't one-hand while charging
+		to_chat(user, span_notice("You flip the switch off as you adjust your grip."))
+		user.visible_message(span_warning("[user] flicks the hammer off!"))
+		charging = FALSE
+
+/obj/item/twohanded/vxtvulhammer/attack(mob/living/carbon/human/target, mob/living/carbon/user) //This doesn't consider objects, only people
+	if (charging) //So you can't attack while charging
+		to_chat(user, span_notice("You flip the switch off before your attack."))
+		user.visible_message(span_warning("[user] flicks the hammer off and raises it!"))
+		charging = FALSE
+	..()
+
+/obj/item/twohanded/vxtvulhammer/AltClick(mob/living/carbon/user)
+	charge_hammer(user)
+
+/obj/item/twohanded/vxtvulhammer/proc/supercharge() //Proc to handle when it's charged for light + sprite
+	supercharged = !supercharged
+	if(supercharged)
+		set_light(2) //Glows when charged
+	else
+		set_light(0)
+	update_icon()
+
+/obj/item/twohanded/vxtvulhammer/proc/charge_hammer(mob/living/carbon/user)
+	if(wielded == FALSE)
```

Comment:
you can just check whether or not the variable is falsey or truthy
if(!wielded)